src/pages/sidepanel/components/TopNavigation.tsx

```diff
@@ -10,7 +10,8 @@ import {
   IconButton,
   Divider,
 } from '@mui/material';
-import { HelpOutline, Settings } from '@mui/icons-material';
+// import { Settings } from '@mui/icons-material';
```

Comment:
Is the setting button intending on making a come back? If not, we should remove commented out code